services/common/src/components/tailings/TailingsSummaryPage.tsx

```diff
@@ -176,17 +183,23 @@ export const TailingsSummaryPage: FC = () => {
                 break;
         }
 
-        history.push(
-            GLOBAL_ROUTES?.EDIT_TAILINGS_STORAGE_FACILITY.dynamicRoute(
-                newTsf?.data.mine_tailings_storage_facility_guid || tsfGuid,
-                mineGuid,
-                newActiveTab || "engineer-of-record",
-                isUserActionEdit
-            )
-        );
-    };
+        if (newActiveTab) {
+            history.push(
+                GLOBAL_ROUTES?.EDIT_TAILINGS_STORAGE_FACILITY.dynamicRoute(
+                    newTsf?.data.mine_tailings_storage_facility_guid || tsfGuid,
+                    mineGuid,
+                    newActiveTab,
+                    isUserActionEdit
+                )
+            );
+        } else {
+            submissionComplete();
+        }
```

Comment:
form navigated to EoR page at the end...
I figure we'll want to do something more complicated at some point when we're done with it, so I made it its own tiny function for now.